### PR TITLE
Create working_with_mltable.ipynb

### PR DESCRIPTION
**Pull request by SturgeonMi**
Update example notebook for MLTable

# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: sdk/assets/data/working_with_mltable.ipynb [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ml import MLClient\n",
```
Comment (ganzhi) [inline sdk/assets/data/working_with_mltable.ipynb:38]: This import fails. It should be:
from azure.ml.ai import MLClient

File: sdk/assets/data/working_with_mltable.ipynb [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    "\n",
+    "## Prerequisites\n",
+    "\n",
+    "You will need to install the following Python package dependencies:\n",
+    "\n",
+    "```bash\n",
+    "pip uninstall azure-ai-ml\n",
+    "pip install --pre azure-ai-ml\n",
+    "pip uninstall mltable",
+    "pip install --pre mltable\n",
```
Comment (ganzhi) [inline sdk/assets/data/working_with_mltable.ipynb:23]: For this notebook to run, we need "pandas" package as well.

File: sdk/assets/data/working_with_mltable.ipynb [hunk 3]
```diff
@@ -0,0 +1,290 @@
+{
+ "cells": [
```
Comment (QianqianNie) [inline sdk/assets/data/working_with_mltable.ipynb:2]: should we also update https://github.com/Azure/azureml-examples/blob/main/sdk/assets/data/data.ipynb?

File: sdk/assets/data/working_with_mltable.ipynb [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import Data, JobInput, CommandJob, Environment\n",
+    "from azure.ai.ml._constants import AssetTypes\n",
+    "\n",
+    "env = Environment(\n",
+    "    image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04\", \n",
+    "    conda_file=\"environments/mltable_environment.yaml\"\n",
```
[the remark this answers is not in the document]
Comment (SturgeonMi) [reply sdk/assets/data/working_with_mltable.ipynb:124]: Updated.

**Review by QianqianNie — APPROVED** [review]

**Review by ganzhi — APPROVED** [review]

**Review by QianqianNie — APPROVED** [review]

**Review by QianqianNie — APPROVED** [review]

**Review by QianqianNie — APPROVED** [review]

**Pull request merged**